percona/controller/pgcluster/controller.go

```diff
@@ -535,6 +539,47 @@ func (r *PGClusterReconciler) reconcileCustomExtensions(cr *v2.PerconaPGCluster)
 		extensionKeys = append(extensionKeys, key)
 	}
 
+	if cr.CompareVersion("2.6.0") >= 0 {
+		var extensionKeysForDeletion []string
+		var extensionsKeysAnnotations []string
+
+		if cr.Spec.Metadata.Annotations != nil {
+			if val, ok := cr.Spec.Metadata.Annotations["custom_extensions_list"]; ok && val != "" {
+				extensionsKeysAnnotations = strings.Split(val, ",")
+			}
+		}
+
+		for i := range extensionsKeysAnnotations {
+			extensionsKeysAnnotations[i] = strings.TrimSpace(extensionsKeysAnnotations[i])
+		}
+		annotationExtensionsMap := make(map[string]struct{})
```

Comment:
Do you need this? Don't see you use at anywhere except to populate it with data.